oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/management/ui/menu/UIMenuManagementService.java

```diff
@@ -83,46 +92,23 @@ public void saveMenuAndStartFetch(List<UIMenuItemSetting> menuItems, int fetchIn
         menu.setConfigurationJson(GSON.toJson(menuItems));
 
         getMenuDAO().saveMenu(menu);
-        startingFetchMenu(fetchInterval);
-    }
-
-    private void startingFetchMenu(int fetchIntervalSecond) {
-        Executors.newSingleThreadScheduledExecutor()
-            .scheduleWithFixedDelay(this, 0, fetchIntervalSecond, java.util.concurrent.TimeUnit.SECONDS);
+        this.menuItemCache.put(true, fetchMenuItems());
```

Comment:
There is no point to call cache `fetchMenuItems`, just call cache.get(true) and the loading cache finds it absent and will call the fetchMenuItems inside the load method

```suggestion
        this.menuItemCache.get(true);
```